### PR TITLE
add mod-description and docs for small button

### PR DESCRIPTION
**Pull request by pochretien**
add a new mod for the p tag
add a new button size for the docs

### CODE REVIEW

File: docs/_components/form_controls/01_input.html [hunk 1]
```diff
@@ -75,6 +75,13 @@
         </label>
         <span class="generic-form-error">Error message goes here!</span>
     </div>
+
+    <div id="description" class="input-field form-group">
+        <h4 class="text-medium-blue">Simple description typography</h4>
+        <p class="mod-description">Lorem ipsum dolor sit amet, interdum est eu. A tellus condimentum hendrerit enim, ligula fusce vitae, leo
```
Comment (JPLachance) [inline docs/_components/form_controls/01_input.html]: Why adding this while it should only be our basic paragraph style? I would simply create a class named `paragraph` (or something else if anyone has a better idea).

Comment (JPLachance) [inline docs/_components/form_controls/01_input.html]: I'm not saying you should set the style right on the `p` tag, I'm saying you should have a class `.paragraph`, just like we have a class `.btn`. `mod-` class is for when you're modifying another existing class.

File: docs/_components/form_controls/01_input.html [hunk 2]
```diff
@@ -75,6 +75,13 @@
         </label>
         <span class="generic-form-error">Error message goes here!</span>
     </div>
+
+    <div id="description" class="input-field form-group">
```
Comment (JPLachance) [inline docs/_components/form_controls/01_input.html]: Why add an id on this div?

File: scss/typography/elements.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+p.mod-description {
```
Comment (JPLachance) [inline scss/typography/elements.scss]: Here I would only do 
```
.paragraph {
  color: $dark-grey;
   font-size: $default-font-size;
   line-height: $big-line-height;
 }
```

File: scss/typography/elements.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+p.paragraph {
```
Comment (vseguin) [inline scss/typography/elements.scss]: I'm wondering if we want to scope this on the p element... what are we usually doing?

File: docs/_components/form_controls/01_input.html (hunk 2 above)
Comment (vseguin) [inline docs/_components/form_controls/01_input.html]: Same question.

File: scss/typography/elements.scss (hunk 4 above)
Comment (JPLachance) [inline scss/typography/elements.scss]: @sroy3 what do you think?

Comment (sroy3) [inline scss/typography/elements.scss]: I'd remove the p. It is useless and makes for harder future changes.

Comment (vseguin) [inline scss/typography/elements.scss]: OMG I WAS RIGHT ON A STYLE COMMENT, IS THIS CHRISTMAS

**Pull request merged**